dotnet/package/common.csproj

```diff
@@ -38,83 +42,75 @@
-    <!-- https://github.com/dotnet/runtime/blob/0647ec314948904319da5eb15e9931f7c85ed1e2/src/installer/pkg/projects/Directory.Build.targets#L281 -->
-    <PropertyGroup Condition="'$(_CreateFrameworkList)' == 'true'">
-      <_FrameworkListFile>$(_packagePath)data/FrameworkList.xml</_FrameworkListFile>
+  <Target Name="_IncludePlatformPackageContent"
+      Condition=" '$(PlatformPackageType)' != '' "
+      BeforeTargets="GetFilesToPackage" >
+    <PropertyGroup Condition=" '$(PlatformPackageType)' == 'TargetingPack' ">
       <_PackTargetPath>ref/net$(BundledNETCoreAppTargetFrameworkVersion)</_PackTargetPath>
       <_PackNativePath>runtimes/$(_RuntimeIdentifier)/native</_PackNativePath>
     </PropertyGroup>
-    <PropertyGroup Condition="'$(_CreateRuntimeList)' == 'true'">
-      <_FrameworkListFile>$(_packagePath)data/RuntimeList.xml</_FrameworkListFile>
+    <PropertyGroup Condition=" '$(PlatformPackageType)' == 'RuntimePack' ">
       <_PackTargetPath>runtimes/$(_RuntimeIdentifier)/lib/net$(BundledNETCoreAppTargetFrameworkVersion)</_PackTargetPath>
       <_PackNativePath>runtimes/$(_RuntimeIdentifier)/native</_PackNativePath>
     </PropertyGroup>
-    <ItemGroup>
-      <_PackageFiles Include="$(_packagePath)$(_PackTargetPath)/*.dll" PackagePath="$(_PackTargetPath)" TargetPath="$(_PackTargetPath)" />
-      <_PackageFiles Include="$(_packagePath)$(_PackNativePath)/*.a" PackagePath="$(_PackNativePath)" TargetPath="$(_PackNativePath)" IsNative="true" />
-      <_PackageFiles Include="$(_packagePath)$(_PackNativePath)/*.dylib" PackagePath="$(_PackNativePath)" TargetPath="$(_PackNativePath)" IsNative="true" />
-      <_FrameworkListFileClass Include="@(_PackageFiles->'%(Filename)%(Extension)')" Profile="$(_PlatformName)" />
+    <ItemGroup Condition=" '$(PlatformPackageType)' == 'TargetingPack' or '$(PlatformPackageType)' == 'RuntimePack' " >
+      <_FxAssembliesToPackage Include="$(_packagePath)$(_PackTargetPath)/*.dll" />
+      <_FxAssembliesDocsToPackage Include="$(_packagePath)$(_PackTargetPath)/*.xml" />
+      <FilesToPackage Include="@(_FxAssembliesToPackage);@(_FxAssembliesDocsToPackage)" TargetPath="$(_PackTargetPath)" />
+      <!-- Always include symbols in runtime packs so that they can be deployed when debugging, otherwise set @(IsSymbolFile) metadata to only include in the symbols.nupkg file -->
+      <FilesToPackage Include="@(_FxAssembliesToPackage->'%(RelativeDir)%(Filename).pdb')" TargetPath="$(_PackTargetPath)" Condition=" '$(PlatformPackageType)' == 'RuntimePack' " />
+      <FilesToPackage Include="@(_FxAssembliesToPackage->'%(RelativeDir)%(Filename).pdb')" TargetPath="$(_PackTargetPath)" IsSymbolFile="true" Condition=" '$(PlatformPackageType)' != 'RuntimePack' " />
+      <NativeRuntimeAsset Include="$(_packagePath)$(_PackNativePath)/*.a" />
+      <NativeRuntimeAsset Include="$(_packagePath)$(_PackNativePath)/*.dylib" />
```

Comment:
With these changes the xamarin/*.h files (such as `_build/Microsoft.iOS.Runtime.iossimulator-arm64.net10.0_18.4/runtimes/iossimulator-arm64/native/xamarin/coreclr-bridge.h` are no longer inlcuded in the runtime packs, I'm not sure if it's ok to omit these? Or if they are required I can restore that.

Reply:
Yes, these header files are required.

It seems our CI is a bit confused because your branch isn't a dev/\* branch, so it ends up posting test results on the commit itself instead of the PR: https://github.com/dotnet/macios/commit/f6502846f4488bced4ba8ed846dbf01eb70680c9#comments

Reply:
Thanks I will restore these and try to keep an eye on the tests on the next commit